src/io/flutter/preview/PreviewView.java

```diff
@@ -715,6 +717,23 @@ public void update(AnActionEvent e) {
   }
 }
 
+class OutlineComponent extends SimpleToolWindowPanel {
```

Comment:
Can you add a brief note about why we're subclassing (in order to proxy through undo/redo to the current active editor)?